src/Concardis/Payengine/Lib/Internal/Constants/CustomerType.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Concardis\Payengine\Lib\Internal\Constants;
+
```

Comment:
There should be only one blank line between the namespace and the class.

Reply:
Remarks from the pull request corrected